features/org.wso2.carbon.kernel.feature/resources/bin/carbon.sh

```diff
@@ -100,6 +101,10 @@ if $mingw ; then
   # TODO classpath?
 fi
 
+#Set the runtimehome based on carbon.sh location
+TEMPCURDIR=`dirname "$PRG"`
+RUNTIME_HOME=`cd "$TEMPCURDIR/.." ; pwd`
```

Comment:
Yes. Here we get the script's location. So the place user run the script doesn't matter